doc/UsersGuide/source/CodeOverview.rst

```diff
@@ -1,14 +1,48 @@
 .. _CodeOverview:
 
 *************************
-Code Overview
+Technical Overview
 *************************
 
+.. _SupportedPlatforms:
+
+=====================================================
+Supported Systems for Running the UFS Weather Model
+=====================================================
+
+Four levels of support have been defined for :term:`UFS` applications, and the UFS Weather Model (:term:`WM`) operates under this paradigm: 
+
+* **Level 1** *(Preconfigured)*: Prerequisite software libraries are pre-built and available in a central location; code builds; full testing of model.
+* **Level 2** *(Configurable)*: Prerequisite libraries are not available in a centralized location but are expected to install successfully; code builds; full testing of model.
+* **Level 3** *(Limited-test platforms)*: Libraries and code build on these systems, but there is limited testing of the model.
+* **Level 4** *(Build-only platforms)*: Libraries and code build, but running the model is not tested.
+
+Level 1 Systems
+==================
+Preconfigured (Level 1) systems for the UFS WM already have the required external libraries available in a central location via :term:`HPC-Stack` or :term:`spack-stack`. The WM is expected to build and run out-of-the-box on these systems, and users can download the WM code without first installing prerequisite software. In general, users must have access to these Level 1 systems in order to use them.
+
```

Comment:
@DeniseWorthen I clarified the wording here so that I explicitly say it runs for Levels 1-3. Level 4 platforms are "supported" but not guaranteed to run (they are "build-only"). We know the WM will build there, and we can help users build it, but we don't guarantee it will run. There has been some discussion of eliminating this category (since what's the point if you can't run it?), but we haven't done that officially yet. 
Normally, we don't mention data in relation to the levels of support because users could be running pretty much any case, not just the sample cases/RT cases we provide. So data for the case they're running may not be available in a central location (or at all) even on a Level 1 system. "Support" has more to do with the availability of the stack and the amount of testing done, according to the definitions passed down to me. However, I can see adding a mention of data for the WM definition, since we're primarily supporting regression testing in standalone mode. I've made that change locally and can push it if you feel strongly that we should include it. 
FYI, there are a few places where levels of support are defined for UFS and/or UFS apps (e.g., [here](https://github.com/ufs-community/ufs/wiki/Supported-Platforms-and-Compilers). "Support" unfortunately hasn't been defined anywhere, but as far as I can tell, it basically means that EPIC will work with you if you fit into one of these levels and are having issues. ;)

Reply:
@gspetro-NOAA I think I don't understand exactly how you're using the term 'support', especially the comment you made that you're _primarily supporting regression testing in standalone mode_. Isn't EPIC's job to support RT testing of the fully UWM repo, including all the non-standalone modes? 

In any case, my original comment was more about implying that on L2-L4, once you build the model OK (however a user manages to do that), it would "run". But it can't run w/o available input files. So to me, you're implying that stuff is there and a user can run once they build.

Reply:
@DeniseWorthen Understood re: the data implications. I've added a mention that data is required. 
Generally speaking, the "Levels of Support" apply specifically to which platforms and compilers we work with (I've clarified this in the section title). We can't _guarantee_ that anything will work on "unsupported" platforms/compilers. 
As to standalone mode, what I meant by "standalone" is “not part of an application workflow,” whereas you were (I think) interpreting "standalone" as "uncoupled." Makes sense, but not what I meant. ;) Gotta love the English language... NLP bots have their work cut out for them. :D